src/ITR/__init__.py

```diff
@@ -2,21 +2,23 @@
 This package helps companies and financial institutions to assess the temperature alignment of investment and lending
 portfolios.
 """
-import pandas as pd
-import numpy as np
 import os
-from .interfaces import EScope
-from . import data  # noqa F401
+
+import numpy as np
+import pandas as pd
 import pint
 
+from . import data  # noqa F401
```

Comment:
We must import `..data` (or however we spell it) before `pint`.  We can import `.interfaces` after `. data`.